README.md

```diff
@@ -295,34 +295,34 @@ We currently have two types of users: **admin user** and **normal user**.
 Our shop's signin logic is mostly handled by Firebase. In our `app.js`, the function onAuthStateChanged is an observer function defined by Firebase that is triggered whenever a user signs in or signs out.
 
 ```javascript
-// Once the user's authentification status is changed
+// Once the user's authentication status is changed
 auth.onAuthStateChanged(async (user) => {
   if (user !== null) {
     const userRef = await createUserProfileDocument(user, [false]);
 ```
 
 Note that in the function, we are calling a function called `createUserProfileDocument`. This function captures the user's information and stores it back to our firebase database, so that next time a user signs in, we are able to verify their identity.
 
-In particular, notice that the second parameter of the function is an array containing a boolean value. The value **determines whether the user passed in will have the admin privelege or not**.
-Normally, the default is false, since we don't want to grant a user the admin privelege.
+In particular, notice that the second parameter of the function is an array containing a boolean value. The value **determines whether the user passed in will have the admin privilege or not**.
+Normally, the default is false, since we don't want to grant a user the admin privilege.
 
-In the case when you want to grant a user privelege (e.x. for your personal account), follow the following steps:
+In the case when you want to grant a user privilege (e.x. for your personal account), follow the following steps:
 
 - make sure you have set up the connection for firebase
 - run the server using `npm run build`
-- change the paramter `[false]` to `[true]` in `createUserProfileDocument` indicate you want the admin privelege for any incoming account registration
-- register a **new** email account (either through google signin or normal signup, but make sure the account doesn't store in the database) using the applicatio:
+- change the parameter `[false]` to `[true]` in `createUserProfileDocument` indicate you want the admin privilege for any incoming account registration
+- register a **new** email account (either through google signin or normal signup, but make sure the account doesn't store in the database) using the application:
 - go to your [**firebase console**](https://console.firebase.google.com/) and make sure your user entry in the firestore collection `users` has the privelege field set to `true`
-- reset the paramter from `[true]` to `[false]` in your `app.js`
+- reset the parameter from `[true]` to `[false]` in your `app.js`
 
-Now, you are able to render components conditionally by checking the user's privelege. For example, like this:
+Now, you are able to render components conditionally by checking the user's privilege. For example, like this:
 
 ```javascript
   <Route
     exact
     path='/admin/add'
     render={() =>
-      !user || !user.privelege ? (
+      !user || !user.privilege ? (
```

Comment:
Should I make a separate issue for changing the React code with the fixed typo?

Reply:
Yes, that would be great! You should want to make a new request for the actual code change.
Thanks neverdue for catching it, I will go ahead and merge it now!